src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();

```

Comment:
Avoid using static variables unless they are constants. In this case, 'random' doesn't need to be static since it's tied to the instance of FigureSupplier.